umaserverdemo/src/main/kotlin/com/lightspark/SendingVasp.kt

```diff
@@ -500,17 +508,25 @@ class Vasp1(
     }
 }
 
-fun Routing.registerVasp1Routes(vasp1: Vasp1) {
+fun Routing.registerSendingVaspRoutes(sendingVasp: SendingVasp) {
     get("/api/umalookup/{receiver}") {
-        call.debugLog(vasp1.handleClientUmaLookup(call))
+        call.debugLog(sendingVasp.handleClientUmaLookup(call))
     }
 
     get("/api/umapayreq/{callbackUuid}") {
-        call.debugLog(vasp1.handleClientUmaPayReq(call))
+        call.debugLog(sendingVasp.handleClientUmaPayReq(call))
     }
 
     post("/api/sendpayment/{callbackUuid}") {
-        call.debugLog(vasp1.handleClientSendPayment(call))
+        call.debugLog(sendingVasp.handleClientSendPayment(call))
+    }
+
+    post("/api/uma/pay_invoice") {
+        call.debugLog(sendingVasp.payInvoice(call))
+    }
+
+    post("/api/uma/request_and_pay_invoice") {
```

Comment:
I think this should really be `request_invoice_payment` or something. It's not actually paying the invoice, but rather, just sending a notification to the user for them to trigger a payment. Feel free to find a name you think fits better. Would be good to align all of the demo vasps with the same routes.